manzi-guev/Broadcaster
# #169916092 Setting the folder structure

### What does this PR do?
- It sets up the project folder structure
### Description of the task to be completed
- Setting up all the project folder structures
- Initializing the project
- Adding all the dependencies needed
- Adding the entry point
### How should this be manually tested?
- Make sure you have Postman
- Clone the project
- do npm i to get all the dependencies
- then do npm run dev
### What are the relevant pivotal tracker stories?
[#169916092](https://www.pivotaltracker.com/story/show/169916092)
### Screenshots(if needed)
